api/resources.js

```diff
@@ -3,9 +3,16 @@ var couchapp = require('couchapp')
   , path = require('path')
   ;
 
-ddoc =  { _id:'_design/bell' }
+endpoint =  { _id:'_design/bell' }
```

Comment:
ddoc is renamed to endpoint, as in api endpoint. This variable contains everything about this api endpoint now including the auth function, the views function, the id, etc.